### PR TITLE
added tiny_json.1.1.6

### PR DESCRIPTION
**Pull request by camlspotter**

### CODE REVIEW

**Comment by rgrinberg** [comment]
Same as my other comment. `jbuilder` should be marked as `{build}` and ocamlfind should be removed. Nevertheless, you've only over-approximated the dependencies so it's still fine to merge as is.

File: packages/tiny_json/tiny_json.1.1.6/opam [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+homepage: "https://camlspotter@bitbucket.org/camlspotter/tiny_json"
+bug-reports:
+  "https://bitbucket.org/camlspotter/tiny_json/issues?status=new&status=open"
+dev-repo: "hg://https://bitbucket.org/camlspotter/tiny_json"
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+depends: [
+  "ocamlfind" {build}
+  "jbuilder"
+]
+available: [ocaml-version >= "4.5.0"]
```
Comment (rgrinberg) [inline packages/tiny_json/tiny_json.1.1.6/opam]: Is this right? Shouldn't it be `4.0.5?`

Comment (camlspotter) [reply packages/tiny_json/tiny_json.1.1.6/opam]: It's 4.05.0, since Bytes is used.

**Comment by camlspotter** [comment]
* ocamlfind is removed from opam
* {build} is added to jbuilder
* ocaml-version constraint is fixed to >= 4.05.0

**Pull request merged**